src/theme/components/pagination.ts

```diff
@@ -15,6 +15,21 @@ export const Pagination = defineVoteWeightTipStyle({
       flexWrap: 'wrap',
       rowGap: '2',
       ml: 'auto',
+
+      '& a': {
+        bgColor: 'transparent !important',
+        color: 'black',
+        borderColor: 'black',
+
+        _hover: {
+          color: 'black',
+        },
+
+        _dark: {
+          color: 'white',
+          borderColor: 'white',
+        },
+      },
```

Comment:
Why are you doing CSS traversing? You have a `PageButton` component you can style on its own.

Also, I don't see how the current page is being differentiated from others (?)